changehc/delphi_changehc/update_sensor.py

```diff
@@ -159,6 +106,12 @@ def __init__(self,
             wip_signal=read_params()["wip_signal"])
         self.updated_signal_names = signal_names
 
+        # initialize members set in shift_dates().
+        self.burnindate = None
```

Comment:
are these supposed to be part of this pr?

Reply:
I was trying to get the linter to pass in every file I touched in this PR.  Updated description to explain.